functions.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+export default_variant="jessie"
```

Comment:
I would prefer to see this move to the `config` file

Reply:
Done!